### PR TITLE
Display larger label when hovered

### PR DESCRIPTION
**Pull request by schloerke**
* adjust max width of end labels to be 1.5 times size
* wrap text to be formatted nicely for hover
* remove white space from short labels when not hovering
* when hovering, a solid white box appears and the label has regular text wrapping
* fix flow issue

screen shot 2019-03-01 at 2 42 16 pm

### CODE REVIEW

File: srcjs/updateGraph/searchString.js [hunk 1]
```diff
@@ -58,7 +58,11 @@ let searchStringWithData = function(obj: Node | Edge | GhostEdge) {
 let searchStringContainer = function(searchElement_: JQuery) {
   searchElement = searchElement_;
   searchElement.on("input", function(e) {
-    searchStringWith($(e.target).val());
+    searchStringWith(
+      $(e.target)
+        .val()
+        .toString()
```
Comment (jcheng5) [inline srcjs/updateGraph/searchString.js]: Why's this? Would `val()` ever return not a string?

File: srcjs/cyto/cytoStyle.js [hunk 2]
```diff
@@ -82,7 +83,7 @@ let graphStyles = {
       "shape-polygon-points": nodeShapes.end.shape,
       width: nodeShapes.end.width,
       height: nodeShapes.end.height,
-      "text-max-width": maxTextWidth,
+      "text-max-width": `${labelWidth * 1.5}px`,
```
Comment (jcheng5) [inline srcjs/cyto/cytoStyle.js]: What's the 1.5 factor for?

File: srcjs/updateGraph/searchString.js (hunk 1 above)
Comment (schloerke) [reply srcjs/updateGraph/searchString.js]: This was a flow issue.  I agree it's always a string, but flow wants to make sure of it.

File: srcjs/cyto/cytoStyle.js (hunk 2 above)
Comment (schloerke) [reply srcjs/cyto/cytoStyle.js]: The ending area usually has long autogenerated labels.  Having it be a little bigger doesn't hurt.  (Tony approved the size bump).

**Pull request merged**